pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -259,15 +262,6 @@ func NewKubeGenericRuntimeManager(
 		"version", typedVersion.RuntimeVersion,
 		"apiVersion", typedVersion.RuntimeApiVersion)
 
-	// If the container logs directory does not exist, create it.
-	// TODO: create podLogsRootDirectory at kubelet.go when kubelet is refactored to
-	// new runtime interface
-	if _, err := osInterface.Stat(podLogsRootDirectory); os.IsNotExist(err) {
-		if err := osInterface.MkdirAll(podLogsRootDirectory, 0755); err != nil {
```

Comment:
We're going from 0755 on the pod log directory to 0750.  Its better, but was it previously 0755 for any reason?

Reply:
I'm not aware of any, other kubelet's directories use `0750`.
But I'm happy to change it back to `0755` if that's an issue.